File: OOPS_LAB/Program_9/time.java
class TimeClass{
    private int hours;
    private int minutes;
    private int second;

    public TimeClass(int hr, int min, int sec) {
        this.hours=hr%24;
        this.minutes=min%60;
        this.second=sec%60;
    }

    public void display()
    {
        System.out.println("Hours : Minutes : Seconds");
        System.out.printf("%02d:%02d:%02d%n", hours, minutes, second);
    }
}

public class time {
    public static void main(String[] args) {
        TimeClass ti1=new TimeClass(14, 32, 30);
        TimeClass ti2 = new TimeClass(6, 50, 36);

        ti1.display();
        ti2.display();
    }
}

File: OOPS_LAB/Program_9/time.cpp
#include<iostream>
#include<print>
using namespace std;

class Time{
    int hours;
    int minute;
    int second;

  public:
    Time(int hr, int min, int sec):hours(hr),minute(min),second(sec){}

    void display()
    {
        println("{:02}:{:02}:{:02}", hours, minute, second);
    }
};

int main(int argc, char const *argv[])
{
    Time t1(2,5,9);
    t1.display();
    return 0;
}